lib/network_utils.ml

```diff
@@ -37,6 +37,20 @@ let bonding_dir = "/proc/net/bonding/"
 let fcoedriver = ref "/opt/xensource/libexec/fcoe_driver"
 let mac_table_size = ref 10000
 
+let filter_map f =
```

Comment:
is there any reason to not use stdext's `Listext.filter_map` ?

Reply:
i did try and add a dependency on `xapi-stdext-std` in the oasis file, but got an error

in the container you get 
```
[planex:networkdbackport local]$ ocamlfind list | grep stdex
stdext              (version: 2.1.0)
```

which made me think that it is not available

Reply:
mb, `Listext` is available and gets opened so we can simply use `List.filter_map`